packages/components/PrefixBadgePicker/PrefixBadgePicker.module.css

```diff
@@ -0,0 +1,137 @@
+.PrefixBadgePicker {
+  display: flex;
+  align-items: center;
+  position: relative;
+  align-self: start;
+  height: 1.5rem;
+}
+
+.PrefixBadgePickerFill {
+  position: absolute;
+  left: -2px;
+  height: 100%;
+  border-radius: 1rem;
+  transition: background-color 0.18s ease-out;
+}
+
+.PrefixBadge {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 1.25rem;
+  height: 1.25rem;
+  flex: 0 0 1.25rem;
+  outline: none;
+}
+
+/*
+  TODO: look into why focus-visible shows when programmatically moving focus
+  https://github.com/w3c/csswg-drafts/issues/5885
+*/
+/* .PrefixBadge:focus-visible {
+  box-shadow: 0 0 0 0.125rem rgba(0, 123, 255, 0.25);
```

Comment:
I need to investigate this more. Currently programatic focus is causing the focus ring to show with the mouse. Alternatively, we could do a scale transition when focused.